client/my-sites/stats/activity-log-banner/upgrade-banner.jsx

```diff
@@ -0,0 +1,62 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { isJetpackSite } from 'state/sites/selectors';
+import Banner from 'components/banner';
+import {
+	PLAN_PERSONAL,
+	PLAN_JETPACK_PERSONAL,
+	FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
+	FEATURE_NO_ADS,
+} from 'lib/plans/constants';
+
+class UpgradeBanner extends Component {
+	render() {
+		const { isJetpack, translate } = this.props;
+		return (
+			<div className="activity-log-banner__upgrade">
+				{ isJetpack ? (
+					<Banner
+						callToAction={ translate( 'Get daily backups' ) }
+						dismissPreferenceName="activity-upgrade-banner-jetpack"
+						event="activity_log_upgrade_click_jetpack"
+						feature={ FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY }
+						list={ [
+							translate( 'See all site activity over the past month' ),
+							translate( 'Rewind your site back to any point' ),
+							translate( 'Automatic threat scanning' ),
+						] }
+						plan={ PLAN_JETPACK_PERSONAL }
+						title={ translate( 'Secure your site with daily backups' ) }
+					/>
+				) : (
+					<Banner
+						callToAction={ translate( 'Upgrade' ) }
+						dismissPreferenceName="activity-upgrade-banner-simple"
+						event="activity_log_upgrade_click_wpcom"
+						feature={ FEATURE_NO_ADS }
+						list={ [
+							translate( 'Get a custom domain name' ),
+							translate( 'Remove WordPress.com ads' ),
+							translate( 'See 30 days of past activity' ),
```

Comment:
Should _"See 30 days of past activity"_ line be first in the list for wpcom nudges and more elaborate than that?

If we don't have demo events visible (https://github.com/Automattic/wp-calypso/pull/25537#issuecomment-398684603), we should make it extra clear that user is seeing upgrade nudge specifically because of the activity log, not just randomly.

Customers might be thinking _"how many days of past activity do I see without a plan?"_ Since they can see Jetpack related events it might not be clear to them that they would see 0 days.

Thoughts?

Reply:
As per @roccotripaldi's comment here — https://github.com/Automattic/wp-calypso/pull/25136#pullrequestreview-128899364 — I believe the banner (for dotcom Simple) will _always_ have the demo activities, they're just being handled in two different PRs.